src/components/Editor/Footer.js

```diff
@@ -41,13 +42,21 @@ type Props = {
 
 class SourceFooter extends PureComponent<Props> {
   prettyPrintButton() {
-    const { selectedSource, togglePrettyPrint } = this.props;
+    const { selectedSource, togglePrettyPrint, isPrettyPrinting } = this.props;
     const sourceLoaded = selectedSource && isLoaded(selectedSource);
 
     if (!shouldShowPrettyPrint(selectedSource)) {
       return;
     }
 
+    if (isPrettyPrinting) {
```

Comment:
could this be selectedSource isLoading and isPrettyPrinted?

Reply:
Yeah, that will be awesome